01-intro/04-flow-control/01-conditionals-and-loops/README.md

```diff
@@ -1,214 +1,209 @@
-# Estructuras condicionales y repetitivas
+# Estruturas condicionais e repetitivas
 
-- Tipo: `lectura`
-- Formato: `self-paced`
-- Duración: `30min`
+- Tipo: `aula prática`
+- Duração: `90min`
```

Comment:
We need to keep this metadata structure (`tipo`, `formato`, `duração`), with the values that @cesmbr added to the parser, that would be like this:

```
- Tipo: `leitura`
- Formato: `individual`
- Duração: `90min`
```

Reply:
Can I use Duração, or I have to use Duración?